app/src/features/mocks/screens/mocksList/components/MocksList/components/MocksTable/hooks/useMocksTableColumns.tsx

```diff
@@ -38,47 +58,81 @@ export const useMocksTableColumns = ({
+              <MdOutlineFolder />
+            </span>
+
+            <Typography.Text ellipsis={true} className="mock-collection-name">
+              {record.name}
+            </Typography.Text>
+
+            <Typography.Text ellipsis={true} className="mock-collection-description">
+              {record.desc}
+            </Typography.Text>
+          </div>
+        ) : (
+          <div className="mock-name-details-container">
+            {isCollection || isRuleEditor ? null : (
+              <Button
+                type="text"
+                onClick={(e) => {
+                  e.stopPropagation();
+                  toggleMockStarAction(record, forceRender);
                 }}
+                className="mock-star-btn"
               >
-                {record?.type === MockType.API ? record.method : record.fileType}
-              </span>
+                <MdOutlineStarOutline className={record.isFavourite ? "starred" : "unstarred"} />
+              </Button>
+            )}
+            <div
+              className="mock-details-container"
+              onClick={(e) => {
+                e.preventDefault();
+                e.stopPropagation();
 
-              <Typography.Text ellipsis={true} className="primary-cell mock-name">
-                {record.name}
-              </Typography.Text>
-            </div>
-            <div className="mock-endpoint">
-              <Typography.Text ellipsis={true}>{"/" + record.endpoint}</Typography.Text>
+                if (!isRecordMockCollection(record)) {
+                  handleNameClick(record.id, record.isOldMock);
+                }
+              }}
+            >
+              <div className="mock-name-container">
+                <span
+                  className="mock-type"
+                  style={{
+                    color:
+                      record?.type === MockType.API
+                        ? REQUEST_METHOD_COLORS[record.method]
+                        : fileTypeColorMap[record.fileType],
+                  }}
+                >
+                  {record?.type === MockType.API ? record.method : record.fileType}
+                </span>
+
+                <Typography.Text ellipsis={true} className="primary-cell mock-name">
+                  {record.name}
+                </Typography.Text>
+              </div>
+              <div className="mock-endpoint">
+                <Typography.Text ellipsis={true}>{"/" + record.endpoint}</Typography.Text>
+              </div>
             </div>
           </div>
         );
       },
     },
     {
+      key: "createdBy",
       title: <div className="rq-col-title">Created by</div>,
       width: 70,
       responsive: ["lg"],
       className: "text-gray",
-      render: (_: any, record: RQMockMetadataSchema) => {
+      render: (_: any, record: RQMockSchema) => {
         return (
           <div className="mock-table-user-icon">
             <UserIcon uid={record.createdBy} />
```

Comment:
createdBy is rendering as Anonymous
![CleanShot 2024-05-01 at 13 09 46](https://github.com/requestly/requestly/assets/16779465/48e0ef96-1e16-4db8-b8d2-346e95e5cdfc)